### DISCUSSION TITLE
Environments approvals create lots of emails that I cannot switch off

### DISCUSSION THREAD

**Discussion by deniszaboronsky** [community/community#37353] [Actions]
According to the documentation it is suggested to use environments to create manual approval steps during CI pipelines. To do this I set the team as the required approver. This functionally works, however the issue is that now the whole team gets an email whenever anyone runs into an approval step. 

With github actions I cannot see a way to change when emails are sent (as opposed to unsubscribing as an individual). This makes email alerts less valuable, so its harder to notice failure alerts.

**Comment by ossanna16** [comment]
Hi there @deniszaboronsky and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
